src/user/model.coffee

```diff
@@ -11,9 +11,12 @@ BLANK_USER =
   name: null
   profile_url: null
   courses: []
+  _course_data: []
+  endpoints: {}
+  isLoaded: false
```

Comment:
dont we do a call to `status/auth` on open, which would pull the endpoints again?